istioctl/cmd/istioctl/kubeinject.go

```diff
@@ -375,7 +375,7 @@ func init() {
 	injectCmd.PersistentFlags().StringVar(&imagePullPolicy, "imagePullPolicy", inject.DefaultImagePullPolicy,
 		"Sets the container image pull policy. Valid options are Always,IfNotPresent,Never."+
 			"The default policy is IfNotPresent.")
-	injectCmd.PersistentFlags().IntVar(&statusPort, "statusPort", inject.DefaultStatusPort,
```

Comment:
I don't think we can make backward incompatible flag changes. 
Keep old flag, add new if needed.

Reply:
This is the same flag. I just refactor the flag name with `inject.StatusPortCmdFlagName` which equals to "statusPort".